packages/docs/pages/checkbox.tsx

```diff
@@ -1,116 +1,158 @@
 import { Checkbox, Form, FormGroup, H1, Panel, Text } from '@bigcommerce/big-design';
 import React, { useState } from 'react';
 
-import { Code, CodePreview, PageNavigation } from '../components';
+import { Code, CodePreview, ContentRoutingTabs, List } from '../components';
 import { CheckboxDescriptionLinkPropTable, CheckboxDescriptionPropTable, CheckboxPropTable } from '../PropTables';
 
 const CheckboxPage = () => {
-  const items = [
-    {
-      id: 'examples',
-      title: 'Examples',
-      render: () => (
-        <>
-          <Panel>
-            <Text>
-              Checkboxes are a stylized <Code>input[type="checkbox"]</Code> with controllable checked/unchecked states.
-            </Text>
-            <CodePreview>
-              {/* jsx-to-string:start */}
-              {function Example() {
-                const [checked, setChecked] = useState(false);
-                const handleChange = () => setChecked(!checked);
+  return (
+    <>
+      <H1>Checkbox</H1>
 
-                return (
-                  <Form>
-                    <FormGroup>
-                      <Checkbox label={checked ? 'Checked' : 'Unchecked'} checked={checked} onChange={handleChange} />
-                      <Checkbox label="Disabled" disabled={true} />
-                    </FormGroup>
-                  </Form>
-                );
-              }}
-              {/* jsx-to-string:end */}
-            </CodePreview>
-          </Panel>
-          <Panel header="Indeterminate">
-            <Text>
-              Checkboxes support <Code primary>isIndeterminate</Code> passed as a prop to show a combined state of
-              partially selected checkboxes.
-            </Text>
+      <Panel header="Overview" headerId="overview">
+        <Text>Checkboxes let users toggle settings on and off within a form.</Text>
+        <Text bold>When to use it:</Text>
+        <List>
+          <List.Item>Use checkboxes when users can make toggle one or more items in a form.</List.Item>
+        </List>
```

Comment:
In hindsight, we probably should have make a component for this 🤔 (just a comment, no action needed)